Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -5476,6 +5476,29 @@ public string GetConsumerDescription(in FileOrDirectoryArtifact artifact)
             return null;
         }
 
+        /// <summary>
+        /// Returns true when associated with a FileOrDirectory artifact has more than 1 consumers.
+        /// </summary>
+        /// <param name="artifact">The artifact for which to find consumer count > 1.</param>
+        /// <returns>The true when consumer count = 0 or 1 or false if consumer count > 1.</returns>
+        [SuppressMessage("Microsoft.Design", "CA1033:InterfaceMethodsShouldBeCallableByChildTypes")]
+        public bool HasMultipleConsumers(in FileOrDirectoryArtifact artifact)

```

Comment:
>HasMultipleConsumers [](start = 20, length = 20)

Per my other suggestion, this method is no longer needed, but for future reference you can get the number of outgoing/incoming edges by calling DataflowGraph.GetOutgoingEdgesCount(nodeId)